src/app/common/services/bootstrap/bootstrap.service.ts

```diff
@@ -18,6 +18,41 @@ export class BootstrapService {
     'http_port': '9000',
     'mqtt_url': 'localhost:1883',
     'edgex_url': 'http://localhost:48090/api/v1/',
+    'nats_url': 'localhost:4222',
+    'export_config': {
+      'exp': {
+        'log_level': 'debug',
+        'nats': 'nats://localhost:4222',
+        'port': '8170',
+      },
+      'mqtt': {
+        'ca_path': 'ca.crt',
+        'cert_path': 'thing.crt',
+        'channel': '35860baf-6660-4f47-a5c5-8eb1bd176175',
+        'host': 'tcp://84.201.171.65:1883',
+        'mtls': false,
+        'password': 'e9c03586-7422-4772-8631-9e0d3f152ca1',
+        'priv_key_path': 'thing.key',
+        'qos': 0,
+        'retain': false,
+        'skip_tls_ver': false,
+        'username': 'b9688074-9bc1-4736-8132-7fd45a0736ef',
```

Comment:
i think that export and agent need to have separate thing id because of mtls

Reply:
Let's think about this and do it in another issue.
When creating gateway - UI will create two things, one of them (main thing) will have in it's metadata written which one is his `export twin`

Reply:
Maybe that will look a bit confusing (two things for one gateway). But if it must be done, let's just do it in another issue.